src/OpenLoco/Scenario.cpp

```diff
@@ -1,13 +1,25 @@
 #include "Scenario.h"
 #include "Graphics/Gfx.h"
 #include "Interop/Interop.hpp"
+#include "Objects/CargoObject.h"
 #include "S5/S5.h"
 #include "Ui/WindowManager.h"
 
 using namespace OpenLoco::Interop;
 
 namespace OpenLoco::Scenario
 {
+    static loco_global<cargo_object*, 0x0050D15C> _50D15C;
+    static loco_global<uint8_t, 0x00526230> objectiveType;
+    static loco_global<uint8_t, 0x00526231> objectiveFlags;
+    static loco_global<uint32_t, 0x00526232> objectiveCompanyValue;
+    static loco_global<uint32_t, 0x00526236> objectiveMonthlyVehicleProfit;
+    static loco_global<uint8_t, 0x0052623A> objectivePerformanceIndex;
+    static loco_global<uint8_t, 0x0052623B> objectiveDeliveredCargoType;
+    static loco_global<uint32_t, 0x0052623C> objectiveDeliveredCargoAmount;
+    static loco_global<uint8_t, 0x00526240> objectiveTimeLimitYears;
+    static loco_global<uint16_t, 0x00526241> objectiveTimeLimitUntilYear;
```

Comment:
Can we turn this into a single objective struct?

Reply:
Yeah but leave for another refactor.